app/ledger.js

```diff
@@ -976,12 +981,19 @@ var getBalance = () => {
 
   ledgerBalance.getBalance(ledgerInfo.address, underscore.extend({ balancesP: true }, client.options),
   (err, provider, result) => {
+    var unconfirmed
+    var info = ledgerInfo._internal.paymentInfo
```

Comment:
just curious why `unconfirmed` needs to be in `ledgerInfo._internal` if it already exists in `ledgerInfo`